app/models/match_decisions/approve_match_housing_subsidy_admin.rb

```diff
@@ -121,15 +139,19 @@ def canceled
         Notifications::MatchCanceled.create_for_match! match
         match.canceled!
       end
+
+      def shelter_declined
+        @decision.class.transaction do
+          @decision.update(status: nil)
+          match.create_confirm_shelter_decline_of_hsa_approval_decision unless match.confirm_shelter_decline_of_hsa_approval_decision.present?
+          match.confirm_shelter_decline_of_hsa_approval_decision.initialize_decision!
+        end
+      end
     end
     private_constant :StatusCallbacks
 
     private def note_present_if_status_declined
-      if note.blank? && status == 'declined'
-        errors.add :note, 'Please note why the match is declined.'
-      end
+      errors.add :note, 'Please note why the match is declined.' if note.blank? && status == 'declined'
```

Comment:
Should this also be true for `shelter_declined`?

Reply:
This doesn't appear to be connected as one would expect.  Declines don't actually have a place to input a note.  But it works as expected.